interchaintest/helpers/manifest.go

```diff
@@ -11,15 +11,15 @@ import (
 	"github.com/strangelove-ventures/interchaintest/v8/ibc"
 )
 
-func ManifestUpdateParams(t *testing.T, ctx context.Context, chain *cosmos.CosmosChain, poaAdmin ibc.Wallet, addressPairs string, automaticInflation bool, coinInflationPerYear sdk.Coin, flags ...string) (sdk.TxResponse, error) {
-	txCmd := []string{"tx", "manifest", "update-params", addressPairs, fmt.Sprintf("%v", automaticInflation), coinInflationPerYear.String()}
+func ManifestUpdateParams(t *testing.T, ctx context.Context, chain *cosmos.CosmosChain, poaAdmin ibc.Wallet, addressPairs string, automaticInflation string, coinInflationPerYear string, flags ...string) (sdk.TxResponse, error) {
```

Comment:
I don't get this since you lose type safety, odd. but sure

Reply:
This allows me to test any invalid input a user could pass on the CLI. Those functions are only used for testing purposes.